django_celery_beat/migrations/0012_auto_20190430_0322.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 2.2 on 2019-04-30 03:22
```

Comment:
thanks for this new migration. I am sorry to let you know that the migrations have changed a lot in master. if you could remove this again and fetch pull from master and regen the new migration?

Reply:
Yup, sorry about that @yywing. there were some crazy issues with past migrations that were being fixed at the same time as your PR, but I think it's been completely sorted out now.  You could maybe even just revert your last commit!

Reply:
It is too hrad to merge it. - - I will regen it.

Reply:
done, hope it will work